distributions/distribution-resources/src/main/resources/conf/html/readme.txt

```diff
@@ -0,0 +1,2 @@
+Check out the openHAB2 documentation for more details:
+http://docs.openhab.org
```

Comment:
@kaikreuzer I have actually no clue what the real purpose of this static content folder is :D
Could you write two sentences?